ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import React from 'react'
+import styled from '../../styles/styled-components'
+
+const CustomElement = styled.div`
+  height: 100%;
+  width: 100%;
+  display: block;
+`
+const Visualization = styled.div`
+  cursor: pointer;
+  opacity: ${props => props.theme.minimumOpacity}
+  :hover {
+    opacity: 1
+  }
+`
+const VisualizationChoice = styled.div`
+    white-space: nowrap;
```

Comment:
How does this pass formatting?

Reply:
I didn't even notice the indentation was a little off, but that's a good question